src/renderer/components/siteinfotools/SiteInfoToolsSection.tsx

```diff
@@ -38,15 +39,21 @@ const SiteInfoToolsSection = ({ site }: Props) => {
 		);
 	}
 
-	return (
-		<div className={styles.SiteInfoToolsSection}>
-			<ToolsHeader site={site} />
-			<ToolsContent
-				className={styles.SiteInfoToolsSection_Content}
-				site={site}
-			/>
-		</div>
-	);
+	return useObserver(() => {
+		const offline = $offline.offline;
```

Comment:
You could destructure here if you wanted
```
const offline = $offline;
```

Reply:
*doh*, I definitely was thinking it was way too many "offline"'s lolol